### PR TITLE
add profile

### PR DESCRIPTION
**Pull request by BrianKiarieMwaniki**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1649"></a>

### CODE REVIEW

File: public/data/briankiariemwaniki.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started OpenSource",
+      "date": "August 2022",
+      "icon": "open-source",
+      "color": "#3bc2cb",
+      "description": "Started freelancing",
+      "url": "https://github.com/EddieHubCommunity"
```
Comment (Panquesito7) [inline public/data/briankiariemwaniki.json:30]: Hmm, not sure if this link should be here unless you're saying you started freelancing on EddieHub. 🤔


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Comment (BrianKiarieMwaniki) [reply public/data/briankiariemwaniki.json:30]: Huh! Makes sense I will edit the file

Comment (BrianKiarieMwaniki) [reply public/data/briankiariemwaniki.json]: Thanks for pointing that out

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉
 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contributions :tada:

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome

**Pull request merged**